eng/Tools.props

```diff
@@ -7,6 +7,9 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.DotNet.Build.Tasks.Packaging" Version="$(MicrosoftDotNetBuildTasksPackagingVersion)" />
+    <!-- enable source link in pkgproj -->
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="$(MicrosoftSourceLinkVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />
```

Comment:
I assume we don't support restore yet on pkgprojs right? If we do I would probably expect to see these two added on the pkgproj targets instead

Reply:
Right, PkgProj doesn't support restore.  Largely because it doesn't behave like an SDK nor consume MS common targets.  I considered doing that work but it was much larger than this tactical fix.

Once we do the work to enable restore in pkgproj (or even eliminate pkgproj in favor of integrating our packaging into CSProj) then we get these references for free since they are part of RepositoryInfo.targets in arcade which is already imported.